skops/io/tests/test_persist.py

```diff
@@ -96,16 +96,16 @@ def wrapper(obj, save_state):
     def debug_get_instance(func):
         # check consistency of argument names and input type
         signature = inspect.signature(func)
-        assert list(signature.parameters.keys()) == ["state", "src"]
+        assert list(signature.parameters.keys()) == ["state", "load_state"]
 
         @wraps(func)
-        def wrapper(state, src):
+        def wrapper(state, load_state):
             assert "__class__" in state
             assert "__module__" in state
             assert "__loader__" in state
-            assert isinstance(src, ZipFile)
+            assert isinstance(load_state, LoadState)
```

Comment:
Can we adjust the `debug_get_state` wrapper to check that each result contains an `"__id__"` key?

Reply:
It looks like that wrapper wraps the `GET_STATE_DISPATCH_FUNCTIONS` and not the high level `get_state` function. 

Given __id__ doesn't get returned from the individual dispatch funcs, it's set in the high level one, I think that check isn't needed.

I've added a check in the`get_instance` wrapper though, so __id__ is being checked during persist

Reply:
Yes, makes sense, thanks